worldedit_commands/init.lua

```diff
@@ -516,9 +516,25 @@ minetest.register_chatcommand("/cylinder", {
 	end, check_cylinder),
 })
 
-minetest.register_chatcommand("/pyramid", {
+local check_pyramid = function(name, param)
+	if worldedit.pos1[name] == nil then
+		worldedit.player_notify(name, "no position 1 selected")
+		return nil
+	end
+	local found, _, axis, height, nodename = param:find("^([xyz%?])%s+([+-]?%d+)%s+(.+)$")
+	if found == nil then
+		worldedit.player_notify(name, "invalid usage: " .. param)

```

Comment:
These messages (this and the `no position` one) should have their first letter capitalized.